packages/account/src/account.ts

```diff
@@ -542,13 +543,15 @@ export class Account extends AbstractAccount {
 
     const findAssetInput = (assetId: string) =>
```

Comment:
We also use a [similar function](https://github.com/FuelLabs/fuels-ts/blob/st/feat/consider-message-with-data-input/packages/account/src/providers/transaction-request/transaction-request.ts#L592) when funding with fake UTXOs, should we just export this function and import it there? Otherwise you would have to update both no?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3083